src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  --is-active: #00acdc;
+  --header-height: 60px;
+
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
don't forget to add more fallback fonts
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/cff5cecd-7624-4602-8a08-b09639863e8d)